src/components/AppHubErrorNoticeBox/AppHubErrorNoticeBox.js

```diff
@@ -0,0 +1,14 @@
+import { NoticeBox } from '@dhis2/ui'
+import React from 'react'
+import i18n from '../../locales/index.js'
+import styles from './AppHubErrorNoticeBox.module.css'
+
+export const AppHubErrorNoticeBox = () => (
+    <NoticeBox
+        error
+        title={i18n.t('Failed to check App Hub for available updates')}
+        className={styles.NoticeBox}
+    >
+        {i18n.t('Your DHIS2 server might be offline.')}
```

Comment:
@cooper-joe should we remove the trailing period (`.`) here?

Reply:
We're also hiding the error details, but they probably add more confusion than clarity